modules/gdscript/tests/scripts/runtime/features/array_string_stringname_equivalent.gd

```diff
@@ -15,10 +15,10 @@ func test():
 	var string_array: Array[String] = []
 	var stringname_array: Array[StringName] = []
 
-	assert(!string_array.push_back(&"abc"))
```

Comment:
`Array.push_back()` returns `void`, so not sure why there's `assert` here. `PackedStringArray` does return a `bool` which is `true` only if it can't resize the vector to add an element, something that likely doesn't matter in this test but I guess it doesn't hurt to check.